gemet/thesaurus/edit_views.py

```diff
@@ -349,3 +350,24 @@ def post(self, request, langcode, id, relation_id):
             foreign_relation.save()
         data = {}
         return self._get_response(data, 'success', 200)
+
+
+class AddNewConceptView(JsonResponseMixin, View):
+
+    def get(self, request, langcode):
+
+        version = Version.objects.create()
+        namespace = Namespace.objects.get(heading=Term.NAMESPACE)
+        new_concept = Term(version_added=version,
+                           namespace=namespace,
+                           status=Term.PENDING)
+        codes = (Concept.objects
+                 .exclude(namespace__heading=InspireTheme.NAMESPACE)
+                 .exclude(code='')
+                 .values_list('code', flat=True))
+        new_code = max(map(int, codes)) + 1
+        new_concept.code = str(new_code).decode("utf-8")
```

Comment:
Why do you need the Unicode value? You get the same result with `unicode(new_code)`.